INFO - 2024-03-12 02:15:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:15:32 --> Attempt to read property "nome" on null
in APPPATH\Modules\Admin\Controllers\Dashboard.php on line 33.
 1 APPPATH\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "nome" on null', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Dashboard.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:15:32 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "nome" on null
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 33)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:16:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:16:09 --> Attempt to read property "nome" on null
in APPPATH\Modules\Admin\Controllers\Dashboard.php on line 33.
 1 APPPATH\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "nome" on null', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Dashboard.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:16:09 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "nome" on null
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 33)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:16:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:16:12 --> Attempt to read property "nome" on null
in APPPATH\Modules\Admin\Controllers\Dashboard.php on line 33.
 1 APPPATH\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "nome" on null', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Dashboard.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:16:12 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "nome" on null
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 33)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:16:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:16:25 --> Attempt to read property "nome" on null
in APPPATH\Modules\Admin\Controllers\Dashboard.php on line 33.
 1 APPPATH\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "nome" on null', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Dashboard.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:16:25 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "nome" on null
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 33)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:16:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:16:27 --> Attempt to read property "nome" on null
in APPPATH\Modules\Admin\Controllers\Dashboard.php on line 33.
 1 APPPATH\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "nome" on null', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Dashboard.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:16:27 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "nome" on null
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 33)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:16:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:16:28 --> Attempt to read property "nome" on null
in APPPATH\Modules\Admin\Controllers\Dashboard.php on line 33.
 1 APPPATH\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "nome" on null', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Dashboard.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:16:28 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "nome" on null
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 33)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:18:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:18:37 --> Attempt to read property "nome" on null
in APPPATH\Modules\Admin\Controllers\Dashboard.php on line 33.
 1 APPPATH\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "nome" on null', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Dashboard.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:18:37 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "nome" on null
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 33)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:18:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:18:39 --> Attempt to read property "nome" on null
in APPPATH\Modules\Admin\Controllers\Dashboard.php on line 33.
 1 APPPATH\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "nome" on null', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Dashboard.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:18:39 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "nome" on null
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Dashboard.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 33)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->index()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:18:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:18:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:18:50 --> Undefined variable $db
in APPPATH\Modules\Admin\DAO\DAOLogin.php on line 20.
 1 APPPATH\Modules\Admin\DAO\DAOLogin.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\DAO\\DAOLogin.php', 20)
 2 APPPATH\Modules\Admin\Controllers\Login.php(29): App\Modules\Admin\DAO\DAOLogin->login('admin', 'admin')
 3 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Login->login()
 4 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:18:50 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined variable $db
#0 C:\wamp64\www\escolar\app\Modules\Admin\DAO\DAOLogin.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\wamp64\\www\\e...', 20)
#1 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Login.php(29): App\Modules\Admin\DAO\DAOLogin->login('admin', 'admin')
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Login->login()
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Login))
#4 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#6 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:19:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:19:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:21:50 --> syntax error, unexpected token "}"
in APPPATH\Modules\Admin\DAO\DAOLogin.php on line 22.
 1 SYSTEMPATH\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\DAO\\DAOLogin.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Admin\\DAO\\DAOLogin')
 3 APPPATH\Modules\Admin\Controllers\Login.php(20): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Admin\\DAO\\DAOLogin')
 4 SYSTEMPATH\CodeIgniter.php(887): App\Modules\Admin\Controllers\Login->__construct()
 5 SYSTEMPATH\CodeIgniter.php(483): CodeIgniter\CodeIgniter->createController()
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:21:50 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
syntax error, unexpected token "}"
#0 C:\wamp64\www\escolar\system\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\e...')
#1 C:\wamp64\www\escolar\system\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Adm...')
#2 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Login.php(20): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Adm...')
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(887): App\Modules\Admin\Controllers\Login->__construct()
#4 C:\wamp64\www\escolar\system\CodeIgniter.php(483): CodeIgniter\CodeIgniter->createController()
#5 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:32:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:32:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:32:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:36:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:36:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:36:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:37:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:37:48 --> Call to undefined method App\Modules\Admin\Models\LoginModel::check_login
in SYSTEMPATH\Model.php on line 873.
 1 APPPATH\Modules\Admin\Controllers\Users.php(30): CodeIgniter\Model->__call('check_login', [])
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:37:48 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(BadMethodCallException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(BadMethodCallException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(BadMethodCallException))
#5 {main}
  thrown
【Previous Exception】
BadMethodCallException
Call to undefined method App\Modules\Admin\Models\LoginModel::check_login
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Users.php(30): CodeIgniter\Model->__call('check_login', Array)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:39:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:41:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:41:15 --> Call to undefined function App\Modules\Admin\Controllers\isLogged()
in APPPATH\Modules\Admin\Controllers\Users.php on line 30.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:41:15 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\isLogged()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:48:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:49:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:49:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:49:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:49:15 --> Call to undefined function App\Modules\Admin\Controllers\isLogged()
in APPPATH\Modules\Admin\Controllers\Users.php on line 30.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:49:15 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\isLogged()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:50:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:50:40 --> Undefined property: stdClass::$username
in APPPATH\Modules\Admin\Controllers\Users.php on line 34.
 1 APPPATH\Modules\Admin\Controllers\Users.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$username', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Users.php', 34)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:50:40 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: stdClass::$username
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Users.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\e...', 34)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:51:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
ERROR - 2024-03-12 02:51:31 --> mysqli_sql_exception: Table 'escolar.adm_funcionarios' doesn't exist in C:\wamp64\www\escolar\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\escolar\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\wamp64\www\escolar\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\wamp64\www\escolar\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\wamp64\www\escolar\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\wamp64\www\escolar\app\Modules\Admin\Models\UserModel.php(43): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Users.php(76): App\Modules\Admin\Models\UserModel->getRecord()
#6 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
#7 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#8 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-12 02:51:31 --> Table 'escolar.adm_funcionarios' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `adm_funcionarios`', [], false)
 2 APPPATH\Modules\Admin\Models\UserModel.php(43): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Modules\Admin\Controllers\Users.php(76): App\Modules\Admin\Models\UserModel->getRecord()
 4 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
 5 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:51:31 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(mysqli_sql_exception), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(mysqli_sql_exception), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 8)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Database\Exceptions\DatabaseException
Table 'escolar.adm_funcionarios' doesn't exist
#0 C:\wamp64\www\escolar\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#1 C:\wamp64\www\escolar\app\Modules\Admin\Models\UserModel.php(43): CodeIgniter\Database\BaseBuilder->get()
#2 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Users.php(76): App\Modules\Admin\Models\UserModel->getRecord()
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
#4 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#5 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:53:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:53:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
ERROR - 2024-03-12 02:53:23 --> mysqli_sql_exception: Table 'escolar.adm_funcionarios' doesn't exist in C:\wamp64\www\escolar\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\escolar\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\wamp64\www\escolar\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\wamp64\www\escolar\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\wamp64\www\escolar\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\wamp64\www\escolar\app\Modules\Admin\Models\UserModel.php(43): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Users.php(76): App\Modules\Admin\Models\UserModel->getRecord()
#6 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
#7 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#8 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-12 02:53:23 --> Table 'escolar.adm_funcionarios' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `adm_funcionarios`', [], false)
 2 APPPATH\Modules\Admin\Models\UserModel.php(43): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Modules\Admin\Controllers\Users.php(76): App\Modules\Admin\Models\UserModel->getRecord()
 4 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
 5 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:53:23 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(mysqli_sql_exception), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(mysqli_sql_exception), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 8)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Database\Exceptions\DatabaseException
Table 'escolar.adm_funcionarios' doesn't exist
#0 C:\wamp64\www\escolar\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#1 C:\wamp64\www\escolar\app\Modules\Admin\Models\UserModel.php(43): CodeIgniter\Database\BaseBuilder->get()
#2 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Users.php(76): App\Modules\Admin\Models\UserModel->getRecord()
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users()
#4 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#5 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 02:56:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:58:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 02:58:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 02:58:44 --> Call to undefined method App\Modules\Admin\Models\LoginModel::check_login
in SYSTEMPATH\Model.php on line 873.
 1 APPPATH\Modules\Admin\Controllers\Usuarios.php(30): CodeIgniter\Model->__call('check_login', [])
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 02:58:44 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(BadMethodCallException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(BadMethodCallException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(BadMethodCallException))
#5 {main}
  thrown
【Previous Exception】
BadMethodCallException
Call to undefined method App\Modules\Admin\Models\LoginModel::check_login
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Usuarios.php(30): CodeIgniter\Model->__call('check_login', Array)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:00:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:00:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:00:27 --> Call to undefined function App\Modules\Admin\Controllers\teste()
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:00:27 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\teste()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:01:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:01:18 --> Call to undefined function App\Modules\Admin\Controllers\teste()
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:01:18 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\teste()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:01:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:01:20 --> Call to undefined function App\Modules\Admin\Controllers\teste()
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:01:20 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\teste()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:01:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:01:21 --> Call to undefined function App\Modules\Admin\Controllers\teste()
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:01:21 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\teste()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:01:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:01:21 --> Call to undefined function App\Modules\Admin\Controllers\teste()
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:01:21 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\teste()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:01:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:01:47 --> Call to undefined function App\Modules\Admin\Controllers\teste()
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:01:47 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\teste()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:13:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:13:04 --> Call to undefined function App\Modules\Admin\Controllers\teste()
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:13:04 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\teste()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:13:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:13:40 --> Call to undefined function App\Modules\Admin\Controllers\teste()
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:13:40 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\teste()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:16:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:16:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:16:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:16:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:16:45 --> Call to undefined function App\Modules\Admin\Controllers\teste()
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 29.
 1 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 2 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:16:45 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Call to undefined function App\Modules\Admin\Controllers\teste()
#0 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:17:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:17:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:17:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:18:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:18:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:18:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:18:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:18:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:18:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:18:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:19:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:19:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:19:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:19:58 --> Undefined property: stdClass::$username
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 34.
 1 APPPATH\Modules\Admin\Controllers\Usuarios.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$username', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\Controllers\\Usuarios.php', 34)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:19:58 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: stdClass::$username
#0 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Usuarios.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\e...', 34)
#1 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:21:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
ERROR - 2024-03-12 03:21:31 --> mysqli_sql_exception: Table 'escolar.adm_funcionarios' doesn't exist in C:\wamp64\www\escolar\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\escolar\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\wamp64\www\escolar\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\wamp64\www\escolar\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\wamp64\www\escolar\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\wamp64\www\escolar\app\Modules\Admin\Models\UserModel.php(43): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Usuarios.php(76): App\Modules\Admin\Models\UserModel->getRecord()
#6 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#7 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#8 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-12 03:21:31 --> Table 'escolar.adm_funcionarios' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `adm_funcionarios`', [], false)
 2 APPPATH\Modules\Admin\Models\UserModel.php(43): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Modules\Admin\Controllers\Usuarios.php(76): App\Modules\Admin\Models\UserModel->getRecord()
 4 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 5 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:21:31 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(mysqli_sql_exception), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(mysqli_sql_exception), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 8)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Database\Exceptions\DatabaseException
Table 'escolar.adm_funcionarios' doesn't exist
#0 C:\wamp64\www\escolar\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#1 C:\wamp64\www\escolar\app\Modules\Admin\Models\UserModel.php(43): CodeIgniter\Database\BaseBuilder->get()
#2 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Usuarios.php(76): App\Modules\Admin\Models\UserModel->getRecord()
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#4 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#5 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:30:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:30:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:30:07 --> Attempt to read property "funcionario_id" on array
in APPPATH\Views\admin\users\list.php on line 41.
 1 APPPATH\Views\admin\users\list.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "funcionario_id" on array', 'C:\\wamp64\\www\\escolar\\app\\Views\\admin\\users\\list.php', 41)
 2 SYSTEMPATH\View\View.php(226): include('C:\\wamp64\\www\\escolar\\app\\Views\\admin\\users\\list.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('admin/users/list', [], true)
 5 APPPATH\Modules\Admin\Controllers\Usuarios.php(79): view('admin/users/list', [...])
 6 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 7 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:30:07 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "funcionario_id" on array
#0 C:\wamp64\www\escolar\app\Views\admin\users\list.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 41)
#1 C:\wamp64\www\escolar\system\View\View.php(226): include('C:\\wamp64\\www\\e...')
#2 C:\wamp64\www\escolar\system\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\wamp64\www\escolar\system\Common.php(1178): CodeIgniter\View\View->render('admin/users/lis...', Array, true)
#4 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Usuarios.php(79): view('admin/users/lis...', Array)
#5 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#6 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#7 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:33:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:33:13 --> Attempt to read property "id" on array
in APPPATH\Views\admin\users\list.php on line 41.
 1 APPPATH\Views\admin\users\list.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "id" on array', 'C:\\wamp64\\www\\escolar\\app\\Views\\admin\\users\\list.php', 41)
 2 SYSTEMPATH\View\View.php(226): include('C:\\wamp64\\www\\escolar\\app\\Views\\admin\\users\\list.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('admin/users/list', [], true)
 5 APPPATH\Modules\Admin\Controllers\Usuarios.php(79): view('admin/users/list', [...])
 6 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 7 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:33:13 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "id" on array
#0 C:\wamp64\www\escolar\app\Views\admin\users\list.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\e...', 41)
#1 C:\wamp64\www\escolar\system\View\View.php(226): include('C:\\wamp64\\www\\e...')
#2 C:\wamp64\www\escolar\system\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\wamp64\www\escolar\system\Common.php(1178): CodeIgniter\View\View->render('admin/users/lis...', Array, true)
#4 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Usuarios.php(79): view('admin/users/lis...', Array)
#5 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#6 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#7 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:37:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:37:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:38:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:38:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
ERROR - 2024-03-12 03:38:45 --> mysqli_sql_exception: Table 'escolar.adm_funcionarios' doesn't exist in C:\wamp64\www\escolar\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\escolar\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\wamp64\www\escolar\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\wamp64\www\escolar\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\wamp64\www\escolar\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\wamp64\www\escolar\app\Modules\Admin\Models\UserModel.php(50): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Users.php(51): App\Modules\Admin\Models\UserModel->editRecord('1')
#6 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users('edit', '1')
#7 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#8 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-12 03:38:45 --> Table 'escolar.adm_funcionarios' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `adm_funcionarios`
WHERE `funcionario_id` = :funcionario_id:', [...], false)
 2 APPPATH\Modules\Admin\Models\UserModel.php(50): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Modules\Admin\Controllers\Users.php(51): App\Modules\Admin\Models\UserModel->editRecord('1')
 4 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users('edit', '1')
 5 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:38:45 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(mysqli_sql_exception), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(mysqli_sql_exception), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 8)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Database\Exceptions\DatabaseException
Table 'escolar.adm_funcionarios' doesn't exist
#0 C:\wamp64\www\escolar\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#1 C:\wamp64\www\escolar\app\Modules\Admin\Models\UserModel.php(50): CodeIgniter\Database\BaseBuilder->get()
#2 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Users.php(51): App\Modules\Admin\Models\UserModel->editRecord('1')
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->users('edit', '1')
#4 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#5 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:38:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:39:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:39:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:39:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:40:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:40:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:41:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:41:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:42:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:42:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:42:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2024-03-12 03:42:11 --> Undefined variable $modelo
in APPPATH\Modules\Admin\DAO\DAOUsuarios.php on line 40.
 1 APPPATH\Modules\Admin\DAO\DAOUsuarios.php(40): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $modelo', 'C:\\wamp64\\www\\escolar\\app\\Modules\\Admin\\DAO\\DAOUsuarios.php', 40)
 2 APPPATH\Modules\Admin\Controllers\Usuarios.php(77): App\Modules\Admin\DAO\DAOUsuarios->getRecord()
 3 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
 4 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 5 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-12 03:42:11 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\escolar\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\escolar\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\escolar\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\e...')
#3 C:\wamp64\www\escolar\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined variable $modelo
#0 C:\wamp64\www\escolar\app\Modules\Admin\DAO\DAOUsuarios.php(40): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\wamp64\\www\\e...', 40)
#1 C:\wamp64\www\escolar\app\Modules\Admin\Controllers\Usuarios.php(77): App\Modules\Admin\DAO\DAOUsuarios->getRecord()
#2 C:\wamp64\www\escolar\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->users()
#3 C:\wamp64\www\escolar\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#4 C:\wamp64\www\escolar\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\wamp64\www\escolar\index.php(79): CodeIgniter\CodeIgniter->run()
#6 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-03-12 03:42:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:42:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:43:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:43:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:43:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:43:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:43:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:43:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:44:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:44:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2024-03-12 03:44:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
